src/main/java/raythx98/duke/task/ToDos.java

```diff
@@ -0,0 +1,78 @@
+package raythx98.duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoField;
+
+import raythx98.duke.exception.DukeException;
+
+/**
+ * Represents a specific task which is a to do.
+ */
+public class ToDos extends Task {
```

Comment:
Should this class be named with a singular noun instead of a plural noun?

Reply:
Agreed, will resolve